sql_csv_utils.py

```diff
@@ -16,7 +16,16 @@ def __init__(self, config, logging_level=logging.INFO):
         self.specify_db_connection = SpecifyDb(db_config_class=self.config)
         self.logger = logging.getLogger("SqlCsvTools")
         self.logger.setLevel(logging_level)
-        self.specify_db_connection.connect()
+        self.check_db_connection()
+
+    def check_db_connection(self):
```

Comment:
checks whether db connection is function in sql_csv_tools at beginning of class init